alena_automation/XPathExercises - Alena2.csv

```diff
@@ -0,0 +1,34 @@
+Xpath Tasks,Authors.xml,Authors2.xml (with namespaces)
+Normal:,,
+all elements,/authors,
+"all ""author"" elements",//author,
+"the root element ""authors"" - 2 variants","/* ,  authors, child::authors",
+"the same as 3,  but this time without using the name of the element",/*,
+all elements enclosed by elements  /authors/author,/authors/author,
+"all ""name"" elements (without using names of parent elements) ",//name,
+all elements which have 1 ancestors,,
+"all ""period"" attributes of ""author"" elements - 2 variants","//author[@period] , ",
```

Comment:
//author[@period] - returns "author" elements that have period attribute, but need those attributes themselves

Reply:
//@period   ?